packages/express/mixins/mixin.core.js

```diff
@@ -46,6 +56,26 @@ class ExpressMixin extends Mixin {
         })
       );
       middlewares.postfiles.push(helmet.noCache());
+      if (typeof this.getLogger === 'function') {
+        const loggerMiddleware = require('../lib/log');
+        app.use(loggerMiddleware(this.getLogger()));
+      }
+    }
+  }
+  inspectServer(server) {
+    if (typeof this.getLogger === 'function') {
+      const logger = this.getLogger();
+      server.on('startup', (address) => {
+        const { basePath = '' } = this.config;
+        const fullAddress = `${address}/${basePath}`;
+        this.serverAddressPromise.resolve(fullAddress);
```

Comment:
Not sure, if we want to resolve to the full URL (with `basePath`) or just `protocol` + `host` + `port`?
/cc @herschel666

Reply:
You are completely right, but I kept the path for logging.